src/components/UploadFile/index.tsx

```diff
@@ -11,6 +11,7 @@ interface OwnPropsTranslations {
     sizesText?: string;
     formatsText?: string;
     tipText?: string;
+    isMobileDevice?: boolean;
```

Comment:
Don path this as props. Use selector here

Reply:
it is a component, and it is not connected to redux